index.html

```diff
@@ -6414,11 +6443,20 @@ <h3>Issuer</h3>
         </p>
 
         <p>
-Relevant metadata about the <code>issuer</code> <a>property</a> is expected
-to be available to the <a>verifier</a>. For example, an <a>issuer</a> can
-publish information containing the public keys it uses to digitally sign
-<a>verifiable credentials</a> that it issued. This metadata is relevant when
-checking the proofs on the <a>verifiable credentials</a>.
+Relevant metadata that is related to the <code>issuer</code> <a>property</a>
+is available to the <a>verifier</a> through the
+<a href="#verify-securing-mechanism">securing mechanism verification
+algorithm</a> as defined in Section <a href="#securing-mechanisms"></a>. This
+information includes the controller, which is typically the `issuer`, of the
+verification method used by the securing mechanism to secure each <a>verifiable
+credential</a>. When verifying a <a>verifiable presentation</a>, the controller
+of the verification method is typically the `holder`.
```

Comment:
I *hope* this edit makes this (single) paragraph somewhat clearer.

I wonder whether the first sentence of this paragraph should discuss ```metadata related to the `holder` property``` in addition to ```metadata related to the `issuer` property```.

It might be better to cover verifiable credentials (and issuers) in one paragraph, and verifiable presentations (and holders) in another.

```suggestion
Metadata related to the `issuer` <a>property</a>
is available to the <a>verifier</a> through the
<a href="#verify-securing-mechanism">verification algorithm of the securing
mechanism</a> as defined in Section <a href="#securing-mechanisms"></a>.
This metadata includes identification of the controller of the verification
method used by the securing mechanism to secure each <a>verifiable credential</a>
or <a>verifiable presentation</a>, of which the controller is typically the
respective `issuer` or `holder`.
```

Reply:
Applied in af271281d8218238c8b03ee014d8d1cb05b4bcba.